lenovo/ideapad/15arh05/default.nix

```diff
@@ -17,9 +17,4 @@
 
   # Cooling management
   services.thermald.enable = lib.mkDefault true;
-
-  # tlp defaults to "powersave", which doesn't exist on this laptop
-  services.tlp.settings = {
```

Comment:
cc @Guekka this removes your tlp configuration. This laptop is from 2020 and hopefully also works well enough with power-profiles-daemon.

Reply:
I guess this should work. Let's go with this and we will revert if I notice an issue, I'm still using this laptop